Added greylisting using postgrey

5ven opened the pull request.

tomav commented on target/postgrey/postgrey.init:
```diff
@@ -0,0 +1,141 @@
+        ;;
+esac
+
+exit $ret
```
Missing newline

tomav commented on target/postgrey/postgrey:
```diff
@@ -0,0 +1,7 @@
+
+POSTGREY_OPTS="--inet=10023"
+
+#POSTGREY_TEXT="Your customized rejection message here"
```
Missing newline

tomav commented on test/tests.bats:
```diff
@@ -79,6 +79,25 @@ load 'test_helper/bats-assert/load'
+  assert_success
+}
+
+#
```
Can you add a test on Postgrey behaviour on incoming mail?

5ven commented:
These new edits are now complete:
- added the missing newlines
- added the support for new parameters: POSTGREY_DELAY, POSTGREY_MAX_AGE, POSTGREY_TEXT
- fixed the issue of non-unlinkable pid when the daemon is stopped, modified the custom init script accordingly
- ensured that the configuration changes are not duplicated on container restart
- updated the tests to support the new parameters
- added two new tests using netcat to ensure that postgrey functions properly

tomav commented:
Restarted build.

tomav commented:
Thanks you @5ven!

5ven commented:
@tomav, you're welcome! This greylisting seems quite useful. I received about 10 spam messages each day to two email addresses but now after setting POSTGREY_DELAY to 3600, the amount of spam went down to 1-2 per day.
